routing/router.go

```diff
@@ -1725,50 +1740,99 @@ func (r *ChannelRouter) SendToRoute(hash lntypes.Hash, route *route.Route) (
 	}
 
 	err := r.cfg.Control.InitPayment(hash, info)
-	if err != nil {
+	switch {
```

Comment:
Is it still possible to do another SendToRoute attempt when a previous one failed with a terminal failure? I think so

Reply:
Yes, possible.